packages/scheduled-tasks/src/lib/structures/ScheduledTaskStore.ts

```diff
@@ -0,0 +1,32 @@
+import { Store } from '@sapphire/pieces';
+import { ScheduledTask } from './ScheduledTask';
+
+export class ScheduledTaskStore extends Store<ScheduledTask> {
+	public readonly repeatedTasks: ScheduledTask[] = [];
+
+	public constructor() {
+		super(ScheduledTask as any, { name: 'scheduled-tasks' });
+	}
+
+	public set(key: string, value: ScheduledTask): this {
+		if (value.interval !== null || value.cron !== null) {
+			this.repeatedTasks.push(value);
+		}
+
+		return super.set(key, value);
+	}
+
+	public delete(key: string): boolean {
+		const index = this.repeatedTasks.findIndex((task) => task.name === key);
+
+		// If the middleware was found, remove it
```

Comment:
```suggestion
		// If the scheduled task was found, remove it
```


This is a copy pasta from API plugin lol